### PR TITLE
Update buildpack to run mithrin script

### PR DESCRIPTION
**Pull request by jonmak08**
- Update scripts to run ./buildpack-run.sh from mithrin repository
- Update README.md formatting
- Update Git Object to point to main branch

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -3,19 +3,8 @@
 
 BUILD_DIR=$1
 
-cd $BUILD_DIR
+NODE_MODULES_CLEANUP_DIR = $BUILD_DIR/devlops/modules/buildpacks/node_modules_cleanup/
```
Comment (jamesthompsongusto) [inline bin/compile:6]: Typo: should be `devops` instead of `devlops` :-)

File: bin/detect [hunk 2]
```diff
@@ -3,9 +3,12 @@
 
 BUILD_DIR=$1
 
-if [ -f $BUILD_DIR/package.json ]; then
-    echo "Node Cleanup"
+NODE_MODULES_CLEANUP_DIR = $BUILD_DIR/devlops/modules/buildpacks/node_modules_cleanup/
```
Comment (jamesthompsongusto) [inline bin/detect:6]: Same typo as above...

**Review by jamesthompsongusto — CHANGES_REQUESTED** [review]

File: bin/compile (hunk 1 above)
Comment (jonmak08) [reply bin/compile:6]: fat fingers LOL!

Comment (jonmak08) [reply bin/compile]: Resolved with https://github.com/ardiustech/heroku-buildpack-selected-node-modules-cleanup/pull/2/commits/1b0e467e364cfce4044dd9d1c9a445afa4dc8ac7

File: bin/detect (hunk 2 above)
Comment (jonmak08) [reply bin/detect]: Resolved with https://github.com/ardiustech/heroku-buildpack-selected-node-modules-cleanup/pull/2/commits/1b0e467e364cfce4044dd9d1c9a445afa4dc8ac7

**Review by jamesthompsongusto — APPROVED** [review]
Looks good! 👍

**Pull request merged**